test/Sleep-test.js

```diff
@@ -91,8 +119,25 @@ describe("Sleep", () => {
       sampleSleepData[5].hoursSlept,
       sampleSleepData[6].hoursSlept
     ]
-    sleep = new Sleep();
-    user1Data = sleep.userSleepData(sampleSleepData, 1);
+    sleepQuality = [
+      sampleSleepData[0].sleepQuality,
+      sampleSleepData[1].sleepQuality,
+      sampleSleepData[2].sleepQuality,
+      sampleSleepData[3].sleepQuality,
+      sampleSleepData[4].sleepQuality,
+      sampleSleepData[5].sleepQuality,
+      sampleSleepData[6].sleepQuality
+    ]
+    sleepQuality = [
+      sampleSleepData[7].sleepQuality,
+      sampleSleepData[8].sleepQuality,
+      sampleSleepData[9].sleepQuality,
+      sampleSleepData[10].sleepQuality,
+      sampleSleepData[11].sleepQuality,
+      sampleSleepData[12].sleepQuality,
+      sampleSleepData[13].sleepQuality
+    ]
+    sleep = new Sleep(sampleSleepData);
```

Comment:
Removed the workaround direction we were using here, this now represents a new Sleep class instantiated with a dataset.